languageserver/integration/commands.go

```diff
@@ -311,243 +372,193 @@ RetryLoop:
+func (i *FlowIntegration) createAccountHelper(conn protocol.Conn) (address flow.Address, err error) {
+	serviceAccount, err := i.project.EmulatorServiceAccount()
+	if err != nil {
+		return flow.Address{}, errorWithMessage(conn, ErrorMessageServiceAccount, err)
 	}
 
-	account, err := i.flowClient.GetAccount(context.Background(), address)
+	signer := serviceAccount.Name()
+
+	defaultKey := serviceAccount.DefaultKey()
+	serviceAccountPrivateKey, err := i.getServicePrivateKey()
 	if err != nil {
-		return nil, nil, err
+		return flow.Address{}, errorWithMessage(conn, ErrorMessageServiceAccountKey, err)
 	}
 
-	if len(account.Keys) == 0 {
-		return nil, nil, fmt.Errorf(
-			"cannot sign transaction: account %s has no keys",
-			address.Hex(),
-		)
+	cryptoKey, err := crypto.DecodePrivateKeyHex(defaultKey.SigAlgo(), serviceAccountPrivateKey)
+	if err != nil {
+		return flow.Address{}, errorWithMessage(conn, ErrorMessagePrivateKeyDecoder, err)
 	}
 
-	accountKey := account.Keys[0]
-	signer := crypto.NewNaiveSigner(privateKey.PrivateKey, privateKey.HashAlgo)
+	keys := []string{cryptoKey.PublicKey().String()}
 
-	return accountKey, signer, nil
-}
+	signatureAlgorithm := defaultKey.SigAlgo().String()
+	hashAlgorithm := defaultKey.HashAlgo().String()
+	var contracts []string
 
-// sendTransactionHelper sends a transaction with the given script, from the
-// currently active account. Returns the hash of the transaction if it is
-// successfully submitted.
-//
-// If an error occurs, attempts to show an appropriate message (either via logs
-// or UI popups in the client).
-func (i *FlowIntegration) sendTransactionHelper(
-	conn protocol.Conn,
-	address flow.Address,
-	tx *flow.Transaction,
-) (flow.Identifier, error) {
-	accountKey, signer, err := i.getAccountKey(address)
+	newAccount, err := i.sharedServices.Accounts.Create(signer, keys, signatureAlgorithm, hashAlgorithm, contracts)
 	if err != nil {
-		return flow.EmptyID, err
+		return flow.Address{}, errorWithMessage(conn, ErrorMessageAccountCreate, err)
 	}
 
-	tx.SetProposalKey(address, accountKey.Index, accountKey.SequenceNumber)
-	tx.SetPayer(address)
+	return newAccount.Address, nil
+}
+
+// storeAccountHelper sends transaction to store account on chain
+func (i *FlowIntegration) storeAccountHelper(conn protocol.Conn, address flow.Address) (newAccount ClientAccount, err error) {
 
-	block, err := i.flowClient.GetLatestBlock(context.Background(), true)
+	serviceAccount, err := i.project.EmulatorServiceAccount()
 	if err != nil {
-		return flow.EmptyID, err
+		return ClientAccount{}, errorWithMessage(conn, ErrorMessageServiceAccount, err)
 	}
 
-	tx.SetReferenceBlockID(block.ID)
+	defaultKey := serviceAccount.DefaultKey()
+	serviceAddress := serviceAccount.Address().String()
+	privateKey := defaultKey.ToConfig().Context["privateKey"]
+
+	// Store new account
+	code := makeManagerCode(transactionAddAccount, serviceAddress)
+	accountAddress := fmt.Sprintf("Address:0x%v", address)
```

Comment:
Why is the address prefixed with `Address:`?

Reply:
This is how we can pass value with `Address` type to a shared lib method.
Shall I refactor it and use JSON-Cadence format like we do for the rest of the code?